### PR TITLE
javascript-operators-and-expressions - Roland Estropia Jr.

### PR DESCRIPTION
**Pull request by restropiajr**
javascript-operators-and-expressions- 1

### CODE REVIEW

File: javascript-operators-and-expressions/javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+// Question 6vi
+const headCount = 10;
+const isSparta = headCount === 300;
+console.log('value of isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+// Question 6vii
+let motto = fullName;
+motto = fullName + ' is the GOAT';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/javascript-operators-and-expressions/main.js]: You should be using the `+=` operator here

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Roland!

File: javascript-operators-and-expressions/javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (restropiajr) [reply javascript-operators-and-expressions/javascript-operators-and-expressions/main.js]: javascript-operators-and-expressions- 2

Added (see picture)

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**